src/prebid.js

```diff
@@ -299,15 +300,19 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id) {
         var width = adObject.width;
         var url = adObject.adUrl;
         var ad = adObject.ad;
-
-        if (doc===document || adObject.mediaType === 'video') {
-          utils.logError('Error trying to write ad. Ad render call ad id ' + id + ' was prevented from writing to the main document.');
+        
+        if (doc === document || adObject.mediaType === 'video') {
+          utils.logError(`Error trying to write ad. Ad render call ad id ${id} was prevented from writing to the main document.`);
         } else if (ad) {
-          doc.write(ad);
-          doc.close();
+          if (isSrcdocSupported(doc)) {
+            doc.defaultView.frameElement.srcdoc = ad;
```

Comment:
@ckbo3hrk 
Here is an example:
Working (filter by "activeview" in network)
http://mjacobson.devnxs.net/~mjacobson/discrepancy_testing/test_page_20.html

Not working:
http://mjacobson.devnxs.net/~mjacobson/discrepancy_testing/test_page_21.html